magma/syntax/sequential2.py

```diff
@@ -333,11 +327,25 @@ def __init__(self, reset_type, has_enable, reset_priority):
         self.has_enable = has_enable
         self.reset_priority = reset_priority
 
+    def _get_or_add_magma_module(self, env: SymbolTable):
+        magma_id = None
+        for k, v in env.items():
+            if v is magma_module:
+                magma_id = k
+                break
+        if magma_id is None:
+            magma_id = gen_free_name(env)
```

Comment:
meant to attach my comment here

Reply:
modulo that lgtm

Reply:
Yea i had a typo in the condition that actually made the code path not run, fixed up the api usage